### DISCUSSION TITLE
note: There is a small bug in the document (Section 2.1.1)

### DISCUSSION THREAD

**Discussion by ChristinaLJC** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#15] [General]
![批注 2022-12-06 145344](https://user-images.githubusercontent.com/44918766/205842052-5a667d30-ae22-410b-b45a-9700861217a3.png)

请注意，在Section 2.1.1有一处小错误需要更正一下。

正确的内容如下：

- \<num of chunks\>: Number of chunks to keep after partition. If this value is set 3 for a \<input file\> of size _**2051**_ KB, it will only keep the first 3 chunks out of 4 chunks for \<index\>. And if it is set 5, it will use 4 instead of 5 because 5 is out of bound. Note that the last **3KiB** that cannot form a chunk will be discarded, all chunks will be 512KB.

Edited:
2563KB -> **2051** KB
3 bytes -> **3 KiB**

**Comment by YanWQ-monad** [comment]
提个小建议，更正的地方可以用 `**` 加粗标出，这样可以方便同学们知道是哪里更正了，而不用手动一个一个词地对照，谢谢。

**Reply by TANSixu** [reply]
感谢建议，已修改！

**Reply by sses7757** [reply]
> Note that the last 3 bytes that cannot form a chunk will be discarded, all chunks will be 512KB.

Shall be

> Note that the last **3 KiB** that cannot form a chunk will be discarded, all chunks will be **512 KiB**.

**Reply by TANSixu** [reply]
> > Note that the last 3 bytes that cannot form a chunk will be discarded, all chunks will be 512KB.
> 
> Shall be
> 
> > Note that the last **3 KiB** that cannot form a chunk will be discarded, all chunks will be **512 KiB**.

Corrected. Thanks!
